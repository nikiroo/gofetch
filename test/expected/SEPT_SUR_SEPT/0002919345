                   LA REINE PAOLA A ÉTÉ RAPATRIÉE                  

  [Têtes Couronnées] 26-09-2018 18:14

  o News link: http://s.7s7.be/3480017
  o Source link: 


    Par: rédaction
    
    26/09/18 - 18h14 Source: Avec Belga    © belga.
    
    MISE  À  JOUR  L'avion  transportant  la reine Paola s'est posé
    mercredi  vers  17h15  à  l'aéroport  militaire  de  Melsbroek.
    L'épouse   d'Albert  II  a  ensuite  été  transférée  vers  les
    cliniques  universitaires  Saint-Luc  à Woluwe-Saint-Lambert, a
    appris Belga à bonne source.
    
    © epa.
    
    Selon  la  RTBF,  la  reine  Paola  a été victime d'un accident
    vasculaire  cérébral  (AVC)  la  nuit dernière alors qu'elle se
    trouvait  à  Venise. Le palais royal avait fait savoir plus tôt
    dans  la  journée  que  la  reine  Paola  serait  rapatriée  en
    Belgique  afin de se soumettre à des examens médicaux. Toujours
    d'après  la  RTBF,  les jours de la reine ne sont pas en danger
    et  son  état  de santé est sous contrôle. Elle est consciente,
    parle  normalement  et  marche, selon nos confrères du Soir. Le
    quotidien  précise que la reine, examinée par «des médecins qui
    la  connaissent  bien, a voyagé dans un avion de la Défense, et
    donc non médicalisé.
    
    Le  Premier ministre Charles Michel, actuellement à New York où
    il  assiste  à  l'assemblée  générale  des Nations unies, lui a
    souhaité  un prompt rétablissement, a indiqué son porte-parole.
    "Le  Premier  ministre s'est renseigné sur son état de santé et
    lui a souhaité un bon rétablissement", a-t-on appris.
    
    Sur  le  site web de la présidence italienne, Sergio Mattarella
    a  fait  part  de sa compassion au roi Philippe. "J'envoie à la
    reine   mes   meilleurs   voeux  de  rétablissement  rapide  et
    complet", peut-on y lire.
    
    © photo news.
    
    © belga.
    
    © belga.



