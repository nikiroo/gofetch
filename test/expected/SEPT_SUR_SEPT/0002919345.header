0La reine Paola a été rapatriée	null/SEPT_SUR_SEPT/0002919345		70
i  [Têtes Couronnées] 26-09-2018 18:14
i
i    L%27avion  transportant  la  reine  Paola s%27est posé mercredi
i    vers  17h15  à  l%27aéroport militaire de Melsbroek. L%27épouse
i    d%27Albert II a ensuite ...
i
