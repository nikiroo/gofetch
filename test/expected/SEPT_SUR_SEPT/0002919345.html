<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story'>
	<h1>La reine Paola a été rapatriée</h1>
	<div class='details'>([Têtes Couronnées] 26-09-2018 18:14)</div>
	<br/>
    <ul>
        <li>News link: <a href='http://s.7s7.be/3480017'>http://s.7s7.be/3480017</a></li>
        <li>Source link: <a href=''></a></li>
    </ul>
	<br/>
	<div class='content' style='text-align: justify'>
		Par: rédaction<br/><br/>26/09/18 - 18h14 Source: Avec Belga    © belga.<br/><br/>MISE À JOUR L&#x27;avion transportant la reine Paola s&#x27;est posé mercredi vers 17h15 à l&#x27;aéroport militaire de Melsbroek. L&#x27;épouse d&#x27;Albert II a ensuite été transférée vers les cliniques universitaires Saint-Luc à Woluwe-Saint-Lambert, a appris Belga à bonne source.<br/><br/>© epa.<br/><br/>Selon la RTBF, la reine Paola a été victime d&#x27;un accident vasculaire cérébral (AVC) la nuit dernière alors qu&#x27;elle se trouvait à Venise. Le palais royal avait fait savoir plus tôt dans la journée que la reine Paola serait rapatriée en Belgique afin de se soumettre à des examens médicaux. Toujours d&#x27;après la RTBF, les jours de la reine ne sont pas en danger et son état de santé est sous contrôle. Elle est consciente, parle normalement et marche, selon nos confrères du Soir. Le quotidien précise que la reine, examinée par «des médecins qui la connaissent bien, a voyagé dans un avion de la Défense, et donc non médicalisé.<br/><br/>Le Premier ministre Charles Michel, actuellement à New York où il assiste à l&#x27;assemblée générale des Nations unies, lui a souhaité un prompt rétablissement, a indiqué son porte-parole. &quot;Le Premier ministre s&#x27;est renseigné sur son état de santé et lui a souhaité un bon rétablissement&quot;, a-t-on appris.<br/><br/>Sur le site web de la présidence italienne, Sergio Mattarella a fait part de sa compassion au roi Philippe. &quot;J&#x27;envoie à la reine mes meilleurs voeux de rétablissement rapide et complet&quot;, peut-on y lire.<br/><br/>© photo news.<br/><br/>© belga.<br/><br/>© belga.
	</div>
<hr/>
</div>
</body>
